infrastructure/lib/atat-web-api-stack.ts

```diff
@@ -134,7 +136,30 @@ export class AtatWebApiStack extends cdk.Stack {
   }
 
   private addTaskOrderRoutes(props?: AtatWebApiStackProps) {
+    // Creates a target server access log bucket shared amongst the Task Order Lifecycle buckets
+    // server access logs enabled on target bucket
+    const taskOrdersAccessLogsBucket = new SecureBucket(this, "taskOrdersLogBucket", {
+      logTargetBucket: "self",
+      logTargetPrefix: "logs/logbucket/",
+      bucketProps: {
+        accessControl: s3.BucketAccessControl.LOG_DELIVERY_WRITE,
+      },
+    });
+    // Error still shows even though the target bucket server access logs is enabled
+    // this suppresses the related cdk-nag for the target bucket error
+    (taskOrdersAccessLogsBucket.bucket.node.defaultChild as s3.CfnBucket).addMetadata("cdk_nag", {
+      rules_to_suppress: [
+        {
+          id: "NIST.800.53-S3BucketLoggingEnabled",
+          reason:
+            "Task orders server access logs are enabled by using the SecureBucket construct. AWS warns against enabling logs on the log bucket -> https://docs.aws.amazon.com/AmazonS3/latest/userguide/enable-server-access-logging.html",
+        },
+      ],
+    });
```

Comment:
So... when we update to CDK Nag 0.0.92 we actually won't need this anymore. Since this is a bug in the current version of nag, I'd like to remove this

Reply:
Okay, I removed it.

Reply:
Removed in 71f6e4a8971f6a87fc95ac7f65f56550d0de3432